test/jdk/sun/tools/jcmd/TestGcCounters.java

```diff
@@ -0,0 +1,41 @@
+import static jdk.test.lib.Asserts.*;
+
+import jdk.test.lib.process.OutputAnalyzer;
+
+/*
+ * @test
+ * @bug 8315149
+ * @summary Unit test to ensure CPU GC hsperf counters are published.
+ * @requires vm.gc.G1
+ *
+ * @library /test/lib
+ *
+ * @run main/othervm TestGcCounters
+ */
+public class TestGcCounters {
+
+    private static final String[] VM_ARGS = new String[] { "-XX:+UsePerfData" };
```

Comment:
The `VM_ARGS` is passed to the jcmd subprocess via `JcmdBase.jcmd()`. This is probably not what we want, we want to pass `-XX:+UsePerfData` to the main process that runs `TestGcCounters.main()`.

We probably need this on line 13:
```
@run main/othervm -XX:+UsePerfData TestGcCounters
```